codespell_lib/data/dictionary.txt

```diff
@@ -2725,7 +2925,9 @@ altogehter->altogether
 altough->although
 altought->although
 altready->already
+altso->also
 alue->value
+alues->values
```

Comment:
Could it be a misspelling of one of the following?
```
aloes, blues, clues, flues, glues,
```

Reply:
I couldn't find any of these.